src/ui/utils/utils-m3u.ts

```diff
@@ -24,6 +24,11 @@ export const parse = (filePath: string): string[] => {
           return acc;
         }
 
+        // Remove trailing carriage returns
+        if (line.charAt(line.length - 1) === '\r') {
+          line = line.substring(0, line.length - 1);
```

Comment:
I think it is better to just split by regex on line 21:
```js
.split(/\r?\n/)
```

Reply:
Great idea!